aldryn_config.py

```diff
@@ -29,20 +29,14 @@ def to_settings(self, data, settings):
             'skin': 'moono-lisa',
         }
 
-        # This could fail if aldryn-django-cms has not been configured yet.
-        boilerplate_name = settings['ALDRYN_BOILERPLATE_NAME']
-
         if data.get('content_css'):
             CKEDITOR_SETTINGS['contentsCss'] = data['content_css']
         else:
             CKEDITOR_SETTINGS['contentsCss'] = ['/static/css/base.css']
 
+        style_set = ''
```

Comment:
You can remove the `if` statement entirely if you use a fallback to your `data.get('style_set')` call:

```suggestion
        style_set = data.get("style_set", "")
```